src/components/Form.js

```diff
@@ -242,7 +245,13 @@ class Form extends Component {
     })
   }
 
-  setAllValues = values => this.props.dispatch(actions.setAllValues(values))
+  setAllValues = (values = {}) =>
+    this.props.dispatch(
+      actions.setAllValues({
+        ...this.props.defaultValues,
```

Comment:
Why did we have this laying them on top of default values again? Does it make sense to remove?

Reply:
I think it makes sense to keep. Default values while providing a `values` prop is still useful in case the `values` prop doesn't contain required default values.